<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderedTableWidget</class>
 <widget class="QWidget" name="OrderedTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="moveUpPushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Move up</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/up.png</normaloff>:/plugins/DsgTools/icons/up.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addPushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Add</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/plus.png</normaloff>:/plugins/DsgTools/icons/plus.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="removePushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Remove</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/remove.png</normaloff>:/plugins/DsgTools/icons/remove.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="moveDownPushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Move down</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/down.png</normaloff>:/plugins/DsgTools/icons/down.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="savePushButton">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/save.png</normaloff>:/plugins/DsgTools/icons/save.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="loadPushButton">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resources.qrc">
        <normaloff>:/plugins/DsgTools/icons/import.png</normaloff>:/plugins/DsgTools/icons/import.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>110</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
